extensions/blocks/opentable/opentable.php

```diff
@@ -41,12 +42,10 @@ function is_available() {
  * registration if we need to.
  */
 function register_block() {
-	if ( is_available() ) {
-		jetpack_register_block(
-			BLOCK_NAME,
-			array( 'render_callback' => 'Jetpack\OpenTable_Block\load_assets' )
-		);
-	}
+	jetpack_register_block(
+		BLOCK_NAME,
+		array( 'render_callback' => 'Jetpack\OpenTable_Block\load_assets' )
```

Comment:
Let's take the opportunity to update this:

```suggestion
		array( 'render_callback' => __NAMESPACE__ . '\load_assets' )
```